omero/sysadmins/index.rst

```diff
@@ -125,7 +125,7 @@ backing-up your installation.
     troubleshooting
     cli/index
     server-backup-and-restore
-    unix/install-web
+    unix/install-web/web-deployment
```

Comment:
This now renders as "web installation" twice, rather than the second being as "web administration"

Reply:
I need to check